spec/features/atom/users_spec.rb

```diff
@@ -7,7 +7,7 @@
     context "user atom feed via private token" do
       it "should render user atom feed" do
         visit user_path(user, :atom, private_token: user.private_token)
-        body.should have_selector("feed title")
+        expect(body).to have_selector("feed title")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.